system-tests/config-files/file_writer_config_static_data.ini

```diff
@@ -2,6 +2,5 @@ hdf-output-prefix="/output-files/"
 command-uri="localhost:9092/TEST_writerCommand"
 status-uri="localhost:9092/TEST_writerStatus"
 status-master-interval=2000
-use-hdf-swmr=false
```

Comment:
Though a small change, it might significant. Can you comment on it?

Reply:
I removed the system test which used the NeXus-Streamer because it is no longer covering anything that the other tests don't cover. This change helps maintains a balance between tests which use SWMR and which don't.

Reply:
Ok.